components/molecules/AnnouncementCard/announcement-card.tsx

```diff
@@ -8,8 +8,13 @@ interface AnnouncementCardProps {
   url: string;
 }
 const AnnouncementCard = ({ title, description, bannerSrc, url }: AnnouncementCardProps) => {
+  const announcementId = `announcement-${Date.now()}`;
```

Comment:
The main change in each file is the main element is an `<aside />` now with an `aria-labelled-by` attribute.